### ISSUE TITLE
Move contextual tips labels away from the normal window subtitle

### ISSUE THREAD

**Issue by maoschanz** [maoschanz/drawing#567]
## Use case

the window subtitle currently covers several use cases:

- wanting an explicit confirmation of what the current tool is (shape/pencil/etc)
- and of its main options (rectangle/dashed/etc.)
- and contextual help for this tool (click on some button to do whatever)
- and contextual tips for this tool (press some modifier to toggle some option)

this is not accessible, as screenreaders are confused by the periodic change

as a user i know it changes, but it's still a pain to wait to have the label i'm looking for

## Suggested solution

near the tool options' popover, there should be a help popover, for AT LEAST the tips

### Possible drawbacks

such a popover has to be hidden on mobile

but mobile user couldn't read long subtitles anyway lol so not a real drawback

### Possible alternatives

no
